lib/omniauth/strategies/openid_connect.rb

```diff
@@ -372,16 +372,18 @@ def id_token_callback_phase
       end
 
       def valid_response_type?
-        return true if params.key?(configured_response_type)
+        return true if configured_response_types.all? { |key| params.key?(key) }
```

Comment:
Should we make sure the value is actually there?

```suggestion
        return true if configured_response_types.all? { |key| params[key].present? }
```